core/docs/spark-qualification-tool.md

```diff
@@ -517,9 +518,9 @@ The report represents the entire app execution, including unsupported operators
 4. _App Duration_: wall-Clock time measured since the application starts till it is completed.
    If an app is not completed an estimated completion time would be computed.
 5. _SQL DF duration_: wall-Clock time duration that includes only SQL-Dataframe queries.
-6. _GPU Opportunity_: wall-Clock time that shows how much of the SQL duration can be accelerated on the GPU.
+6. _GPU Opportunity_: wall-Clock time that shows how much of the SQL duration and ML functions(if applicable) can be accelerated on the GPU.
 7. _Estimated GPU Duration_: predicted runtime of the app if it was run on GPU. It is the sum of the accelerated
-   operator durations along with durations that could not run on GPU because they are unsupported operators or not SQL/Dataframe.
+   operator durations and ML functions durations(if applicable) along with durations that could not run on GPU because they are unsupported operators or not SQL/Dataframe.
```

Comment:
We need to update the ToolTips of the UI too.

Reply:
Thank you! I wasn't aware of this. Updated it for Gpu Opportunity where applicable. PTAL.